deepomatic/cli/frame.py

```diff
@@ -2,21 +2,27 @@
 
 
 class Frame(object):
-    def __init__(self, name, filename, image, video_frame_index=None):
+    def __init__(self, name, filename, image, video_frame_index=None, absolute_video_frame_index=None):
         # The Frame object is used as a data exchanged in the different queues
         self.name = name  # name of the frame
         self.filename = filename  # the original filename from which the frame was extracted
         self.image = image  # an opencv loaded image (numpy array)
-        self.video_frame_index = video_frame_index  # index of the frame in the video sequence
+        self.video_frame_index = video_frame_index  # index of the frame in the output video sequence
```

Comment:
Maybe use `decoded_video_frame_index` as well here.
```suggestion
        self.decoded_video_frame_index = decoded_video_frame_index  # index from the decoded frames (skipped frames are not included)
```